### DISCUSSION TITLE
iOS JS Support

### DISCUSSION THREAD

**Discussion by JohnTsaii** [alibaba/GaiaX#404] [Q&A]
我有一些JS支持的问题需要讨论一下：
1. 有JS相关文档么
2. iOS 看源码不支持QuickJS, 为什么不支持，苹果审核原因？
3. 关于native module 实现。目前支持哪些函数（setStyle，setProps 都是空实现）

**Comment by JohnTsaii** [comment]
@ronghui1219 @biezhihua @jingcheng1988

**Reply by biezhihua** [reply]
@JohnTsaii JS目前的主要开发者是 @ronghui1219 具体需要他来回答了。

周一会将你这块的问题转达给他。

**Comment by kobeLw** [comment]
为啥我按demo运行的，代码一直都不走js的生命周期，断点调试也不行？
